devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -0,0 +1,267 @@
+
+def off() {
+	onOffCmd(0x00, 1)
+}
+
+def refresh() {
+	refreshCmd(1)
+}
+
+def ping() {
+	refresh()
+}
+
+def resetCmd(endpoint = null) {
+	log.debug "Resetting endpoint: ${endpoint}"
+	delayBetween([
+			encap(zwave.meterV3.meterReset(), endpoint),
+			encap(zwave.meterV3.meterGet(scale: 0), endpoint),
+			"delay 500"
+	], 500)
+}
+
+def reset() {
+	resetCmd(1)
+}
+
+def childOn(deviceNetworkId) {
+	def switchId = deviceNetworkId?.split("/")[1] as Integer
+	sendHubCommand onOffCmd(0xFF, switchId)
+}
+
+def childOff(deviceNetworkId) {
+	def switchId = deviceNetworkId?.split("/")[1] as Integer
+	sendHubCommand onOffCmd(0x00, switchId)
+}
+
+def childRefresh(deviceNetworkId) {
+	def switchId = deviceNetworkId?.split("/")[1] as Integer
+	sendHubCommand refreshCmd(switchId)
+}
+
+def childReset(deviceNetworkId) {
+	def switchId = deviceNetworkId?.split("/")[1] as Integer
+	sendHubCommand resetCmd(switchId)
+}
+
+private refreshAll() {
+	childDevices.each { childRefresh(it.deviceNetworkId) }
+	sendHubCommand refresh()
+}
+
+private resetAll() {
+	childDevices.each { childReset(it.deviceNetworkId) }
+	sendHubCommand reset()
+}
+
+def installed() {
+	log.debug "Installed ${device.displayName}"
+	response(zwave.multiChannelV3.multiChannelEndPointGet().format())
+}
+
+def configure() {
+	def cmds = [
+			encap(zwave.configurationV2.configurationSet(parameterNumber: 255, size: 1, configurationValue: [0])), 	// resets configuration
+			encap(zwave.configurationV2.configurationSet(parameterNumber: 4, size: 1, configurationValue: [1])),	// enables overheat protection
+			encap(zwave.configurationV2.configurationSet(parameterNumber: 80, size: 1, configurationValue: [2])),	// send BasicReport CC
+			encap(zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 2048)), 	// enabling kWh energy reports on ep 1
+			encap(zwave.configurationV1.configurationSet(parameterNumber: 111, size: 4, scaledConfigurationValue: 600)),	//... every 10 minutes
+			encap(zwave.configurationV1.configurationSet(parameterNumber: 102, size: 4, scaledConfigurationValue: 4096)), 	// enabling kWh energy reports on ep 2
+			encap(zwave.configurationV1.configurationSet(parameterNumber: 112, size: 4, scaledConfigurationValue: 600)), 	//... every 10 minutes
+			encap(zwave.configurationV1.configurationSet(parameterNumber: 90, size: 1, scaledConfigurationValue: 1)),	//enables reporting based on wattage change
+			encap(zwave.configurationV1.configurationSet(parameterNumber: 91, size: 2, scaledConfigurationValue: 20)) 	//report any 20W change
+	]
+	delayBetween(cmds) + "delay 1000" + resetAll() + refreshAll()
+}
+
+private encap(cmd, endpoint = null) {
+	if (endpoint) {
+		zwave.multiChannelV3.multiChannelCmdEncap(destinationEndPoint:endpoint).encapsulate(cmd).format()
+	} else {
+		cmd.format()
+	}
+}
+
+private addChildSwitches(numberOfSwitches) {
+	for(def endpoint : 2..numberOfSwitches) {
+		String childDni = "${device.deviceNetworkId}/$endpoint"
```

Comment:
Hey, so we've been trying to be a bit more consistent with how we handle child DNIs and on the zigbee side we have settled on "<deviceDNI>:<endpoint>" so using the colon as a separator.  Could you change this to be the same so we can try to have consistency across devices.